core/src/utilities/helpers/navigation-helpers.js

```diff
@@ -556,6 +556,39 @@ class NavigationHelpersClass {
     }
     return res;
   }
+
+  getAllViewGroupSettings() {
+    return LuigiConfig.getConfigValue('navigation.viewGroupSettings');
+  }
+
+  getViewGroupSettings(viewGroup) {
+    const viewGroupSettings = this.getAllViewGroupSettings();
+    if (viewGroup && viewGroupSettings && viewGroupSettings[viewGroup]) {
+      return viewGroupSettings[viewGroup];
+    } else {
+      return {};
+    }
+  }
+
+  findViewGroup(node, originalNode) {
+    if (node.viewGroup) {
+      if (originalNode && originalNode !== node) {
+        if (
+          node.viewUrl &&
+          originalNode.viewUrl &&
+          IframeHelpers.getLocation(node.viewUrl) === IframeHelpers.getLocation(originalNode.viewUrl)
+        ) {
+          return node.viewGroup;
+        }
+
+        return undefined;
+      } else {
+        return node.viewGroup;
+      }
+    } else if (node.parent) {
+      return this.findViewGroup(node.parent, originalNode || node);
+    }
+  }
```

Comment:
Could you add some JSDoc comments onto these 3 functions . I think since you worked with them here, you can explain them even better.